pkg/verify/signature.go

```diff
@@ -142,6 +143,11 @@ func verifyEnvelopeWithArtifact(verifier signature.Verifier, envelope EnvelopeCo
 	if err != nil {
 		return fmt.Errorf("could not verify artifact: unable to extract statement from envelope: %w", err)
 	}
+	err = limitSubjects(statement)
```

Comment:
style nit, can make this one clause with:

```
if err := limitSubjects(statement); err != nil {
  return err
}
```

Reply:
I'm surprised we don't have a linter complaining about that.